gemrb/core/WorldMap.h

```diff
@@ -94,8 +96,8 @@ class GEM_EXPORT WMPAreaEntry {
 	ieStrRef LocCaptionName = ieStrRef::INVALID;
 	ieStrRef LocTooltipName = ieStrRef::INVALID;
 	ResRef LoadScreenResRef;
-	ieDword AreaLinksIndex[4]{};
-	ieDword AreaLinksCount[4]{};
+	EnumArray<WMPDirection, int> AreaLinksIndex;
+	EnumArray<WMPDirection, int> AreaLinksCount;
```

Comment:
Why the change of type, doesn't it just create more work for us in the importer?

Reply:
a type change had to occur somewhere and the compiler complained about `GUIScript` first, which uses `int` :)

I don't know what is preferable, AFICT it doesn't matter which type we use (`int` vs `ieDword`).

Reply:
I'd prefer to keep matching the "spec", which uses ieDword.